be/src/service/service_be/lake_service.cpp

```diff
@@ -200,6 +202,7 @@ void LakeServiceImpl::publish_version(::google::protobuf::RpcController* control
                          << " txn_id=" << request->txn_ids()[0];
             std::lock_guard l(response_mtx);
             response->add_failed_tablets(tablet_id);
+            st.status().to_protobuf(response->mutable_status());
```

Comment:
```suggestion
            st.to_protobuf(response->mutable_status());
```

Reply:
Thanks, sorry for the mistake, my local docker just also encountered this error, I've fixed.